### PR TITLE
update assertion td-context-ns-thing-mandatory

### PR DESCRIPTION
**Pull request by sebastiankb**
with the option that can also carry the TD 1.0 namespace

also see https://github.com/w3c/wot-thing-description/issues/1324

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/pull/1339.html" title="Last updated on Jan 12, 2022, 4:32 PM UTC (0efbfd0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/1339/7d14b4c...0efbfd0.html" title="Last updated on Jan 12, 2022, 4:32 PM UTC (0efbfd0)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1201,15 +1201,17 @@ <h2>Core Vocabulary Definitions</h2>
...
+          <code>https://www.w3.org/2019/wot/td/v1</code>
+          <em class="rfc2119" title=
+          "MUST">MUST</em> be the first entry and the <code>https://www.w3.org/2022/wot/td/v1.1</code> <em class="rfc2119" title=
+          "MUST">MUST</em> be the second entry.</span>
           <span class="rfc2119-assertion" id=
           "td-context-ns-thing-optional">When <code>@context</code>
           is an <a href="#dfn-array" class="internalDFN"
           data-link-type="dfn">Array</a>, the anyURI
-          <code>https://www.w3.org/2022/wot/td/v1.1</code> <em class=
+          <code>https://www.w3.org/2019/wot/td/v1</code> <em class=
```
Comment (danielpeintner) [inline index.html:1214]: I think this change was unintended, right?

Comment (sebastiankb) [reply index.html]: thanks. Should be fixed now

**Review by benfrancis — APPROVED** [review]
Thank you for fixing this!

**Review by danielpeintner — APPROVED** [review]

**Review by mlagally — APPROVED** [review]

**Comment by sebastiankb** [comment]
from today's TD call, decided to merge.

@mlagally I like to discuss the issue #1341 when you also be available

**Pull request merged**